portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,81 @@
+body {
+  margin: auto;
+  margin-bottom: 20px;
+  text-align: center;
+  font-family: 'B612', sans-serif;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  text-align: center;
+  margin: auto;
+  margin-top: 65px;
+  width: 90%;
+  border-bottom: 2px solid black;
+}
+
+/* https://www.w3schools.com/howto/howto_js_navbar_sticky.asp */
+.navbar {
+  overflow: hidden;
+  background-color: black;
+  /*Make the navbar sticky*/
+  position: fixed;
+  top: 0;
+  width: 100%;
+  text-align: center;
+}
+
+.navbar a {
+  display: inline-block;
+  color: white;
+  text-align: center;
+  padding: 20px;
+  text-decoration: none;
 }
 
-#greeting-container {
+.navbar a:hover {
+    color: greenyellow;
```

Comment:
spacing issue (this is not important for this time, but we'll start caring when we review PRs during the core project)